src/tips/en/use-another-language.md

```diff
@@ -17,4 +17,10 @@ And then in DevTools:
 * Open the settings (press F1 or click the cog icon).
 * Click the "Match browser language" checkbox.
 
-![Screenshot of the settings panel in Edge showing a checkbox to match devtools with the browser language](/assets/img/use-another-language.png)
\ No newline at end of file
+Alternatively, you can only change the language of the DevTools using the following steps,
+* Open DevTools Settings (F1 or Shift+?)
+* Preferences > (Under Appearance) Language > Select a language > Reload DevTools
```

Comment:
I try to always use bold for UI strings. So anything that appears in the UI should be in bold. In this sentence this means Preferences should be bold for example. Here is how I would do it:

```suggestion
* **Preferences** > **Appearance** > **Language**. Select a language and then click **Reload DevTools**.
```